apps/application/serializers/common.py

```
@@ -213,7 +214,7 @@ def append_chat_record(self, chat_record: ChatRecord):
                      chat_user_id=self.chat_user_id, chat_user_type=self.chat_user_type,
                      asker=self.get_chat_user()).save()
             else:
-                QuerySet(Chat).filter(id=self.chat_id).update(update_time=datetime.now())
+                QuerySet(Chat).filter(id=self.chat_id).update(update_time=timezone.now())
             # 插入会话记录
             chat_record.save()
             ChatCountSerializer(data={'chat_id': self.chat_id}).update_chat()
```

The code looks generally correct and well-structured according to the provided context. However, here's a few minor suggestions and optimizations:

1. **Use `QuerySet.first()`**: It might be more efficient to retrieve a single instance of `Chat` instead of using `QuerySet`.
   ```python
   queryset = QuerySet(Chat).filter(id=self.chat_id)
   if queryset.exists():
       chat_instance = queryset.first()
       chat_instance.update_time = timezone.now()
       chat_instance.save()
   ```

2. **Consider Using `get_object_or_404()`:** If you want to raise an exception when no object is found, this can help manage errors gracefully.
   ```python
   from django.shortcuts import get_object_or_404

   chat_instance = get_object_or_404(QuerySet(Chat), id=self.chat_id
   chat_instance.update_time = timezone.now()
   chat_instance.save()
   ```

These changes make the code slightly more robust and potentially more performant in certain scenarios depending on the load and environment it will run in.